api/middleware/capturebody.go

```diff
@@ -0,0 +1,12 @@
+package middleware
+
+import (
+	"bytes"
+	"io/ioutil"
+)
+
+func CaptureBody(c *Context) {
+	body, _ := ioutil.ReadAll(c.Req.Request.Body)
```

Comment:
this seems like a potential error should be handled somehow so we don't pass an "invalid" body into the request handlers